i18n/zh/docusaurus-plugin-content-docs/current/setup/install-with-keadm.md

```diff
@@ -138,29 +201,67 @@ Example:
 1. `--cloudcore-ipport` 是必填参数。
 2. 加上 `--token` 会自动为边缘节点生成证书，如果您需要的话。
 3. 需要保证云和边缘端使用的 KubeEdge 版本相同。
-4. 加上 `--with-mqtt` 会自动为边缘节点以容器运行的方式部署 `mosquitto` 服务
 
 输出：
 
 ```shell
-Host has mosquit+ already installed and running. Hence skipping the installation steps !!!
 ...
-KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
+KubeEdge edgecore is running, For logs visit: journalctl -u edgecore.service -xe
+```
+
+您也可以使用 `systemctl status edgecore` 来检查edgecore是否正确运行。
+```shell
+# systemctl status edgecore
+● edgecore.service
+   Loaded: loaded (/etc/systemd/system/edgecore.service; enabled; vendor preset: enabled)
+   Active: active (running) since Wed 2022-10-26 11:26:59 CST; 6s ago
+ Main PID: 2745865 (edgecore)
+    Tasks: 13 (limit: 4915)
+   CGroup: /system.slice/edgecore.service
+           └─2745865 /usr/local/bin/edgecore
 ```
 
 > 也可以使用 `keadm beta join` 来添加边缘节点。
 
+#### keadm deprecated join
+
+您可以使用 `keadm deprecated join` 来启动EdgeCore的release版本. 它会从 [KubeEdge release website](https://github.com/kubeedge/kubeedge/releases) 下载并作为系统服务启动。
+
+举个例子:
+
+```shell
+keadm deprecated join --cloudcore-ipport="THE-EXPOSED-IP":10000 --token=27a37ef16159f7d3be8fae95d588b79b3adaaf92727b72659eb89758c66ffda2.eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTAyMTYwNzd9.JBj8LLYWXwbbvHKffJBpPd5CyxqapRQYDIXtFZErgYE --kubeedge-version=1.12.0
+```
+
+输出:
+
+```shell
+MQTT is installed in this host
+...
+KubeEdge edgecore is running, For logs visit: journalctl -u edgecore.service -xe
+```
+
+### 部署边缘节点演示
+
+Refer to the [Deploy demo on edge nodes](./install-with-binary#deploy-demo-on-edge-nodes) documentation.
```

Comment:
cannot find this file `deploy-demo-on-edge-nodes`

Reply:
It is in English version, now I have translated and added this part into Chinese version.